.github/ISSUE_TEMPLATE/false_positive.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug Report (False Positive)
+about: Create a bug report about a lint warning that is wrong for Clippy
```

Comment:
This hasn't changed. Rebase/squash gone wrong? :thinking:

Reply:
ah, I modified the false negative example accidentally ... :rofl: